src/main.rs

```diff
@@ -271,10 +271,10 @@ mod parse_dependency_tests {
     }
 
     #[test]
-    fn dependency_with_minor_version() {
+    fn dependency_with_compatible_version() {
         assert_eq!(
-            parse_dependency("anyhow==1.1.0"),
-            Dependency::CrateIo("anyhow".to_string(), Some("=1.1.0".to_string()))
+            parse_dependency("anyhow=^1.1.0"),
+            Dependency::CrateIo("anyhow".to_string(), Some("^1.1.0".to_string()))
         )
     }
```

Comment:
@yozhgoor this is a pointless test because the regex has a catch-all `.+`. You can remove it entirely

Reply:
@eopb can i ask you to remove it ?

Reply:
I know how to do it from here now :+1: